packages/beta/src/__tests__/api/alertsApi.int.spec.ts

```diff
@@ -201,3 +201,44 @@ test('sort alerts', async () => {
   });
   expect(response.items.length).toBeGreaterThan(0);
 });
+
+test('cursor pagination', async () => {
+  const client: CogniteClient = setupLoggedInClient();
+  const response = await client.alerts.list({
+    limit: 1,
+    sort: {
+      property: 'createdTime',
+      order: 'desc',
+    },
+    cursor: '',
+  });
+  expect(response.items.length).toBe(1);
+
+  // create 1001 alerts
+  await client.alerts.create(
+    Array.from({ length: 1001 }, (_, i) => ({
+      source: 'smth',
+      channelExternalId: 'test_channel',
+      externalId: `test_alert_${i}`,
+    }))
+  );
+
+  const alerts = client.alerts
+    .list({
+      sort: {
+        property: 'createdTime',
+        order: 'desc',
+      },
+      cursor: response?.nextCursor,
```

Comment:
question(blocking): should this cursor be used from previous response ?
I don't think we need to pass cursor when we're autoPagingToArray.